2024-01-16 06:44:29.171 -05:00 [INF] API: / Time: 85 ms
2024-01-16 06:49:07.846 -05:00 [INF] API: / Time: 124 ms
2024-01-16 06:49:08.614 -05:00 [INF] API: / Time: 219 ms
2024-01-16 06:49:21.446 -05:00 [ERR] a4a8da53-c729-4c43-b993-9d826f954867: One or more validation errors occurred.
TektonLabs.TechnicalTest.Infraestructure.Exceptions.BusinessValidationException: One or more validation errors occurred.
   at TektonLabs.TechnicalTest.Core.Extensions.ValidatorExtensions.ValidateOrThrowValidationExceptionAsync[T](IValidator`1 instance, T dto, CancellationToken cancellation) in C:\APPS\Oiga.Courses.TechnicalTest\TektonLabs.TechnicalTest\TektonLabs.TechnicalTest.Core\Extensions\ValidatorExtensions.cs:line 18
   at TektonLabs.TechnicalTest.Core.Services.ProductService.Create(CreateProductDto ProductDto) in C:\APPS\Oiga.Courses.TechnicalTest\TektonLabs.TechnicalTest\TektonLabs.TechnicalTest.Core\Services\ProductService.cs:line 86
   at TektonLabs.TechnicalTest.Api.Controllers.ProductsController.Create(CreateProductDto model) in C:\APPS\Oiga.Courses.TechnicalTest\TektonLabs.TechnicalTest\TektonLabs.TechnicalTest.Api\Controllers\ProductsController.cs:line 33
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at TektonLabs.TechnicalTest.Infraestructure.Exceptions.ErrorHandlingMiddleware.Invoke(HttpContext context) in C:\APPS\Oiga.Courses.TechnicalTest\TektonLabs.TechnicalTest\TektonLabs.TechnicalTest.Infraestructure\Excepcions\ErrorHandlingMiddleware.cs:line 31
2024-01-16 06:49:21.794 -05:00 [INF] API: Products/Create Time: 3621 ms
